### DISCUSSION TITLE
download blackarch tools on arch

### DISCUSSION THREAD

**Discussion by e1a3g3l7e** [BlackArch/blackarch#4376] [Support]
After i did : 

# Run https://blackarch.org/strap.sh as root and follow the instructions.

$ curl -O https://blackarch.org/strap.sh
$ echo 76363d41bd1caeb9ed2a0c984ce891c8a6075764 strap.sh | sha1sum -c
$ chmod +x strap.sh
# Run strap.sh
$ sudo ./strap.sh
$ sudo pacman -Syu

i've got everything right i guess and after typing those : 

$ sudo pacman -Sgg | grep blackarch | cut -d' ' -f2 | sort -u
$ sudo pacman -S blackarch-<category>
$ sudo pacman -Sg | grep blackarch

no answers in my terminal, just nothing 
![image](https://github.com/user-attachments/assets/dd271d50-067b-42ed-9480-9eb4e5cccd61)

**Comment by e1a3g3l7e** [comment]
PS: wdgaf for being in that path i tried normally before yk

**Comment by D3vil0p3r** [comment]
What are you trying to do by `sudo pacman -S blackarch-`. This command won't work.

By the way, run and post here the output of `cat /etc/pacman.conf` and `sudo pacman -Syy`.

**Comment by noraj** [comment]
What are you trying to achieve? Listing available tools?
